### ISSUE TITLE
Exporter is EXTREMELY SLOW when the account being backupped has 2FA

### ISSUE THREAD

**Issue by dxdvostfr** [Tyrrrz/DiscordChatExporter#992]
### Version

2.37.1

### Flavor

GUI (Graphical User Interface), CLI (Command Line Interface)

### Export format

HTML

### Details

I tried exporting a single DM channel with the tool and I've been astonished by the slowness of the exporter. (I'm talking 48h+ hours for 50k messages). 
I think it even got stuck beacuse it wasn't moving at all past ~50%.

I've tried using the CLI, the GUI, I switched to the second-to-last version to make sure it was not a new bug, I used a different token, nothing worked. 

Then I thought about 2FA, disabled it, and retried, everything went fast and smoothly.

I think if this issue cannot be fixed (it may be unfixable because of how discord handles request with a 2FA account), the tool or the wiki should at least mention we should disable 2FA when using the tool, or it will be too slow to be usable.

### Steps to reproduce

-Toggle 2FA on your discord account
-Get your fresh token
-Try to export any chat using the tool

**Comment by Tyrrrz** [comment]
I doubt it has anything to do with 2FA. Are you exporting a channel that has a lot of bot messages with mentions, by any chance? If not, can you try if version `2.36.4` is any faster?

**Comment by dxdvostfr** [comment]
Maybe it is something else but I doubt it. During the whole process I was trying to export the same conversation, using the same options. I closed the tool and retried multiple times, the same result. But as soon as I removed 2FA (and used the new token) it immediatly worked.

Before that I had already tried using 2.36,4 because I saw you changed someting with rate limits. It had no impact, just like using a new token with 2FA still on.
